### PR TITLE
Get medicare and marketplace data migrated from node.js app. This is …

### PR DESCRIPTION
**Pull request by gortonz**
…a intermediate submit and there may be a few data and import issues need to be fixed

### CODE REVIEW

File: lib/data/dibp-data.json [hunk 1]
```diff
@@ -1,171 +1,177 @@
...
-          { "label": "2016-05", "value":  47 },
-          { "label": "2016-06", "value":  42 }
-      ]
-    },
+        {
+            "id": "device-type-03",
+            "name": "Desktop",
+            "recorded_at": "2016-06-30T01:01:01.111Z",
+            "units": "i",
+            "note": "note for this dataset",
```
Comment (elisechant) [inline lib/data/dibp-data.json]: @mibraheim looks like this probably is lorem ipsum from forever ago?

**Comment by tobyhede** [comment]
That last merge commit merges the branch into itself, I don't think that was the intention?

File: spec/serializers/widget_serializer_spec.rb [hunk 2]
```diff
@@ -41,4 +42,9 @@
     end
   end
 
+  context 'with options' do
```
Comment (elisechant) [inline spec/serializers/widget_serializer_spec.rb]: with_options?

File: app/serializers/widget_serializer.rb [hunk 3]
```diff
@@ -2,20 +2,31 @@ class WidgetSerializer < ActiveModel::Serializer
...
 
   def definition
     object.description
   end
 
+  def displayRoundedData?
+    object.options && object.options.key?('displayRoundedData')
+  end
+
   def displayRoundedData
```
Comment (elisechant) [inline app/serializers/widget_serializer.rb]: any reason we have displayRoundedData? and displayRoundedData   ?

Comment (tobyhede) [inline app/serializers/widget_serializer.rb]: One is a check to see if it exists

Comment (tobyhede) [inline app/serializers/widget_serializer.rb]: The other is the field. If you scroll up, its how we make the value conditional ... we add it to the attribute declartion

**Pull request merged**